src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -354,7 +354,7 @@ export class JupyterNotebookBase implements INotebook {
         return deferred.promise;
     }
 
-    public inspect(code: string, cancelToken?: CancellationToken): Promise<JSONObject> {
+    public inspect(code: string, offsetInCode = 0, cancelToken?: CancellationToken): Promise<JSONObject> {
```

Comment:
shall we put this param to the end, so no breaking to exsiting code?

Reply:
In this case, there are 2 concerns:
1. The interface will be different to `getCompletion`, which also calls jupyter kernel.
2. It's hard to provide enough information for hover request, for example: the code is "module_func()", while the cursor is at '', if we simply truncate the string, we can only get "module_f", which cannot get anything. Or we need to infer which word is required, which is complicated and break the design of jupyter inspect API.